content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -1,117 +1,168 @@
+## Configuration Settings
 
-- The [configuration settings](#configuration-settings) that affect resource utilization
-- The [network and hardware](#network-and-hardware) requirements to achieve consistent, good performance across the XRP Ledger network
+Ripple recommends using these configuration guidelines to optimize resource utilization and performance of your `rippled` server.
 
-## Configuration Settings
+You can set the following parameters in the `rippled.cfg` file used for your `rippled` server. You can access an example configuration file, `rippled-example.cfg`, in the [`cfg` directory](https://github.com/ripple/rippled/blob/develop/cfg/rippled-example.cfg) in the `rippled` GitHub repo.
 
-Ripple recommends the following guidelines to improve performance. You can set the following parameters in the `rippled.cfg` file to improve performance for your `rippled` server.
 
 ### Node Size
 
-The `node_size` parameter determines the size of database caches. Larger database caches decrease disk I/O requirements at a cost of higher memory requirements. Ripple recommends you always use the largest database cache your available memory can support. See the following table for recommended settings.
+Set the `node_size` based on your server's expected load and the amount of memory you can make available to `rippled`.
+
+Ripple recommends you always use the largest node size your available RAM can support. See the following table for recommended settings.
 
 #### Recommendation
 
-| Available RAM for `rippled` | `node_size` value | Notes                              |
-|:----------------------------|:------------------|:-----------------------------------|
-| < 8GB                       | `tiny`            | Not recommended                    |
-| 8GB                         | `small`           |                                    |
-| 16GB                        | `medium`          |                                    |
-| 32GB                        | `huge`            | Recommended for production servers |
+Each `node_size` has a corresponding requirement for available RAM. For example, if you set `node_size` to `huge`, you should have at least 32GB of available RAM to help ensure that `rippled` can run smoothly.
+
+To tune your server, it may be useful to start with `tiny` and increase the size to `small`, `medium`, and so on as you refine the requirements for your use case.
+
+| `node_size` value | RAM available for `rippled` | Notes                      |
+|:------------------|:----------------------------|:---------------------------|
+| < 8GB             | `tiny`                      | Not recommended for testing or production servers. If not specified in `rippled.cfg`, `tiny` is the default value. |
+| 8GB               | `small`                     | Recommended for test servers. |
+| 16GB              | `medium`                    | Note that the `rippled-example.cfg` has its `node_size` value set to `medium`. |
```

Comment:
Nit: change the notes to active voice (also remove the word "note" as the column header already says they're notes!):

> The default `rippled.cfg` sets this value.

Reply:
Thank you for this! Made a few changes to address -- I hope they work for you.